tests/phpunit/modules/saved_searches/class.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests for Hm_Saved_Searches class
+ * Comprehensive coverage of both search types (simple and advanced):
+ */
+class Hm_Test_Saved_Searches_Class extends TestCase {
+
+    private $saved_searches;
+    private $sample_data;
+    private $advanced_search_data;
+
+    public function setUp(): void {
+        require_once __DIR__.'/../../bootstrap.php';
+        require_once APP_PATH.'modules/core/modules.php';
+        require_once APP_PATH.'modules/saved_searches/modules.php';
```

Comment:
These three lines are inappropriately placed here. They should be part of the test suite bootstrap process.